queries/account-data.rq

```diff
@@ -0,0 +1,22 @@
+SELECT DISTINCT
+  ?issuerLabel
+  ?property
+  # if multiple usernames for the same platform have the same rank we merge them into a comma seperated list
+  (GROUP_CONCAT(DISTINCT ?url; separator=",") AS ?url_list)
+WHERE {
+  ?property wdt:P31 wd:Q105388954 ;
+            wikibase:directClaim ?wdt ;
+            wdt:P1630 ?formatter ;
+            # we use issuer label instead of property description, to just get the platform name
+            wdt:P2378 ?issuer . # Wikidata item of this property?
+
+  wd:{{.}} ?wdt ?value .
+
+  BIND(IRI(CONCAT(SUBSTR(?formatter, 1, STRLEN(?formatter) -2), ?value)) AS ?url)
```

Comment:
Instagram seems broken. Debug by:
1. Running this query in QDQS-GUI with a sane param
2. Clear cache (`snowman cache account-data.rq --invalidate`)

Reply:
Fixed: https://www.wikidata.org/w/index.php?title=Property:P2003&diff=prev&oldid=1455598361